web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -36,14 +37,22 @@ export const SidebarCardsList = (props: Props): ReactElement => {
     }
   };
 
-  const showEmptyState = (): boolean => {
+  const showEmptyOpportunitiesMsg = (): boolean => {
     return (
       props.displayCertifications &&
       props.displayFundings &&
```

Comment:
```suggestion
      (props.displayCertifications || props.displayFundings) &&
```
This could happen here or later. It will need tests.

I was thinking about how these messages are kindof inverse cases and it made me want to check if there is a time we display certs and not fundings (or vice versa). There is, in `FORMED_AND_REGISTERED`. In that instance, if a user hides all their opportunities they will currently see nothing. This is a bug, not caused by your work, but if you wanted to fix it, it would be quick.

Reply:
@jp, once a user is in `formed and registered` they see the funding nudge card and if they hide all their certs, they will still see them in the hidden section (under the accordion). I don't think this is an issue.